modules/api/go.mod

```diff
@@ -86,25 +86,29 @@ require (
 	kubevirt.io/api v1.2.2
 	kubevirt.io/containerized-data-importer-api v1.59.0
 	sigs.k8s.io/controller-runtime v0.19.0
-	sigs.k8s.io/controller-tools v0.14.0
+	sigs.k8s.io/controller-tools v0.16.1
 	sigs.k8s.io/yaml v1.4.0
 )
 
+replace k8c.io/kubermatic/v2 => /Users/waleedmalik/src/ahmedwaleedmalik/kubermatic
```

Comment:
Will update this once https://github.com/kubermatic/kubermatic/pull/13644 is merged